src/hotspot/share/runtime/mutexLocker.hpp

```diff
@@ -58,7 +58,7 @@ extern Monitor* CodeSweeper_lock;                // a lock used by the sweeper o
 extern Mutex*   MethodData_lock;                 // a lock on installation of method data
 extern Mutex*   TouchedMethodLog_lock;           // a lock on allocation of LogExecutedMethods info
 extern Mutex*   RetData_lock;                    // a lock on installation of RetData inside method data
-extern Monitor* VMOperationQueue_lock;           // a lock on queue of vm_operations waiting to execute
+extern Monitor* VMOperation_lock;                // a lock on queue of vm_operations waiting to execute
 extern Monitor* VMOperationRequest_lock;         // a lock on Threads waiting for a vm_operation to terminate
```

Comment:
Can the declaration of VMOperationRequest_lock be removed now too? Since it's no longer being defined in mutexLocker.cpp

Reply:
Fixing, pushing later.